cluster-classification/readme.md

```diff
@@ -0,0 +1,11 @@
+# Global Field Importance for Clusters
```

Comment:
perhaps we can also include this one in the top-level metadata.json as a component? and surely to the top-level readme.md in the examples list?